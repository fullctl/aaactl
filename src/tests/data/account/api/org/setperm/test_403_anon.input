{
  "client": "api_client_anon",
  "data": {
    "id": 4,
    "component": "billing.contact",
    "perms": "crud"
  }
}
