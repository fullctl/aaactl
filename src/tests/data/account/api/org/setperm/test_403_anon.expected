{
  "status": 403,
  "response": {"data": [], "errors": {"detail": "Authentication credentials were not provided."}, "profiling": {"queries": 0}}
}
